package.json

```diff
@@ -99,6 +99,7 @@
   },
   "jupyterlab": {
     "extension": true,
-    "outputDir": "jupyter_theme_editor/labextension"
+    "outputDir": "jupyter_theme_editor/labextension",
```

Comment:
In this file, you need to edit `files` array to include the schema:

```diff
    "files": [
      ...
+     "schema/*.json"
    ],
```

Reply:
I have add the file in the `package.json` file.